src/microbe_stage/organelle_components/MovementComponent.cs

```diff
@@ -58,12 +65,16 @@ protected override void CustomAttach()
     /// <summary>
     ///   Calculate the momentum of the movement organelle based on
     ///   angle towards middle of cell
+    ///   If the flagella is  placed in the microbe's center, hence delta equals 0,
+    ///   consider defaultPos as the organelles "false" position.
     /// </summary>
-    private static Vector3 CalculateForce(Hex pos, float momentum)
+    private static Vector3 CalculateForce(Hex pos, float momentum, Vector3 defaultPos)
     {
-        Vector3 organelle = Hex.AxialToCartesian(pos);
+        Vector3 organellePosition = Hex.AxialToCartesian(pos);
         Vector3 middle = Hex.AxialToCartesian(new Hex(0, 0));
-        var delta = middle - organelle;
+        var delta = middle - organellePosition;
+        if (delta == Vector3.Zero)
+            delta = defaultPos;
```

Comment:
Same here. By the way, the codes are very similar so it could be worth having some math util, here making a difference unless it is zero.

Reply:
I dont think thats worth doing for  2 lines of code.